src/templates/base/2019/table_of_contents.html

```diff
@@ -121,6 +121,12 @@ <h2 class="part-name">{{ self.appendix() }}</h2>
       </section>
     </div>
 
+    <section class="ebookd-download">
+      {% if ebook_exists(lang, year) %}
+      <p><a href="/static/pdfs/web_almanac_{{ year }}_{{ lang }}.pdf">{{ self.ebook_download() }}</a></p>
```

Comment:
Should this use `url_for`?

Reply:
Can we use that for non-template files?

Which leads to another question for you - should this PDF live in static folder or templates folder? It's not dynamically generated as such which is why I put in static folder but @mikegeyser originally had it generating in templates folder. If it's in the static folder then don't need Python to serve it so seems faster to me. Especially as it's a big file so that's why I went with the static folder.

What's your opinion?

Reply:
Yeah that can be used for anything with an `@app.route` decorator. So if we pass it the year and lang it will dynamically generate URL corresponding to this pattern `@app.route('/<lang>/<year>/ebook')`.

Reply:
ebook.html should be a template
ebook.pdf should be a static file

Reply:
So this line is a `.pdf` file, so in static folder, so not `@app.route` so fine to leave as is?

Reply:
And it is a link to the PDF and not the `ebook.html` as think the PDF is more useful as per my comments here: https://github.com/HTTPArchive/almanac.httparchive.org/pull/817#issuecomment-627028323

Reply:
You're right, this is ok.